README.md

```diff
@@ -1,2 +1,37 @@
 # Socks Proxy
-A Socks5 server that, when used in conjunction with a [viproxy](https://github.com/blocky/viproxy), grants networking capabilities to a program running in a Nitro Server. The current iteration, however, is coded to only allow connections to [Let's Encrypt](https://letsencrypt.org/) for the purpose of getting self-signed certificates. If you wish to communicate with other services, you will have to update the `allowedAddrs` or `allowedFQDNs` variables.
\ No newline at end of file
+
+A Socks5 server that, when used in conjunction with a
+[viproxy](https://github.com/blocky/viproxy), grants networking capabilities to
+a program running in a Nitro Server.
+
+The current iteration, however, is coded
+to either just allow connections to [Let's Encrypt](https://letsencrypt.org/) for the
+purpose of getting self-signed certificates, or it allows all connections.
+
+## Usage
```

Comment:
It was non-trivial to figure out how to get started doing local development with this tool, so I added some info to the readme.

Also... maybe this subsection should be called something else.  Any suggestions?

Reply:
thanks!